# Game status constants
uint8 GAME_STARTING=0
uint8 GAME_STARTED=1
uint8 GAME_PAUSED=2

int64 score
int32 current_ball_count
int32 total_ball_count
uint8 game_status
float32 status_time_left
bool robot_is_safe